### ISSUE TITLE
Question: The accuracy is very low

### ISSUE THREAD

**Issue by KP-Zhang** [sherlockhoatszx/TimeSeriesPredctionUsingDeeplearning#1]
Hi, Sir,
Your work is very impressive. I am quite new to keras. When I run your code, I add a parameter -- metrics=['accuracy'] in the compile function. But the acc is unreasonably low. I do not know why. What happened? Can you help me out of this confusion?
Thank you for your time.

**Comment by sherlockhoatszx** [comment]
hi Dylan:
	My accuracy metrics (predict the next 2 days) will be around 85-93%.  I have upgrade my code by adding one Convolution1D layers before feed into the GRUorLSTM layers. You can try to use Convolution1D before it, and fine tune  your hyper parameter .Hope that can help.

**Comment by KP-Zhang** [comment]
Hi sherlockhoatszx,
Thank you for your reply.
Forgive me stupid question. I am not quite sure whether I apply a right way to calculate the 'accuracy'.
If I simply modify your code: `model.compile(loss='mean_squared_error', optimizer=Adam())` **TO**
`model.compile(loss='mean_squared_error', optimizer=Adam(), metrics=['accuracy']),` does it work?
How do you calculate your accuracy metrics?
**Happy Lantern Festival!**

**Comment by KP-Zhang** [comment]
Hi Sherlockhoatszx,
Do you have a moment?
model.compile(loss='mean_squared_error', optimizer=Adam()) TO
model.compile(loss='mean_squared_error', optimizer=Adam(), metrics=['accuracy']), does it work?
How do you calculate your accuracy metrics?

**Comment by sherlockhoatszx** [comment]
I use MAPE to calculate it . MAPE function was wrote by myself.

**Comment by KP-Zhang** [comment]
Thank you for your reminding. I am trying to do MAPE based on this site [http://stats.stackexchange.com/questions/58391/mean-absolute-percentage-error-mape-in-scikit-learn/62511#62511](url). Thank you for your time.

**Comment by KP-Zhang** [comment]
Hi Sherlockhoatszx, it is always helpful to read your code. I have some confusion related to Time Series Forecasting.
There is traffic data (1440 pieces in total, and 288 pieces each day) I collected to predict traffic flow. The data is collected every 5 min in five consecutive working days. I am going to use the traffic data of the first four day to train the prediction model, while the traffic data of the fifth day is used to test the model.
Here is my question, if I want to predict the traffic flow of the fifth day, do I only need to treat my prediction as one-step forecast or do I have to predict 288-step?
Look forward to your advice.
Thanks for your post again.
